sound.js

```diff
@@ -119,36 +119,49 @@ Sound.prototype.stop = function(callback) {
 
 Sound.prototype.setStreamType = function(streamType, callback) {
   // For android only.
-  // resets current media player, sets streamType, and prepare again.
```

Comment:
I removed `reset()` in here! I thought it is more convenient that `setStreamType` includes `reset()`. But, it makes additional logic to send reset result to be needed. Thus, I remove `reset()` and make it raise error when it is called before `reset()`.